batchflow/models/tf/deep_galerkin.py

```diff
@@ -433,9 +528,95 @@ def _callable(shifted_time):
 
         return _callable
 
+
     def predict(self, fetches=None, feed_dict=None, **kwargs):
         """ Get network-approximation of PDE-solution on a set of points. Overloads :meth:`.TFModel.predict` :
         `solution`-tensor is now considered to be the main model-output.
         """
         fetches = 'solution' if fetches is None else fetches
         return super().predict(fetches, feed_dict, **kwargs)
+
+
+
+class DGSolver():
```

Comment:
Perhaps, we could make it compatible with BaseModel API to use this class in Pipelines as well